client/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Three version numbers inside __coincidentally__ happen to be one number. Is it really coincidence? Do you plan on using different numbers as versions? If not, use `<properties>` and put project version there.